common.blocks/popup/popup.tests/gemini.blocks/test-nested/test.styl

```diff
@@ -1 +0,0 @@
-../../simple.blocks/test/test.styl
```

Comment:
It is very difficult. It will be necessary to rewrite all the code around the `enb-bem-examples`.